packages/create-sitecore-jss/src/initializers/nextjs/index.ts

```diff
@@ -50,6 +56,15 @@ export default class NextjsInitializer implements Initializer {
       addInitializers = addInitAnswer.addInitializers;
     }
 
+    if (!addInitializers.includes('nextjs-styleguide')) {
```

Comment:
Should we also check `args.templates` for 'nextjs-styleguide'? e.g. if they used `--templates nextjs,nextjs-styleguide`

Reply:
Good point, I missed it